gui/MainWindow.cxx

```diff
@@ -3572,7 +3572,11 @@ void MainWindow::computeCamera()
   auto landmarks = d->groundControlPointsHelper->registrationLandmarks();
 
   // Set up algorithm
-  auto config = kv::config_block::empty_config();
+  auto config = readConfig("resection_camera.conf");
```

Comment:
for consistency, we should call this file `gui_resection_camera.conf`.  That naming convention may change in the future, but for now, all config files that are read directly from the TeleSculptor GUI via hard-coded file names in the GUI have a "gui_" prefix.

Reply:
added gui_ prefix